addons/finger/functions/fnc_incomingFinger.sqf

```diff
@@ -28,7 +28,7 @@ private _fingerPos = if (_sourceUnit == ACE_player) then {
 TRACE_3("incoming finger:", _sourceUnit, _fingerPosPrecise, _fingerPos);
 
 private _data = [diag_tickTime, _fingerPos, ([_sourceUnit, false, true] call EFUNC(common,getName))];
-[GVAR(fingersHash), _sourceUnit, _data] call CBA_fnc_hashSet;
+GVAR(fingersHash) set [hashValue _sourceUnit, _data];
```

Comment:
Nice idea, I didn't think of that.
hashValue of object is basically BIS_fnc_netID that works in SP and MP